initializers/smart-agent-core.js

```diff
@@ -16,6 +16,7 @@
 
 'use strict'
 const { Initializer, api } = require('actionhero')
+const { createDefaultRuntime } = require('@evan.network/api-blockchain-core')
```

Comment:
(belongs to question group 1)
❓ Does the seed need its own runtime? Derived smart agents will have a running runtime and the seed itself will most probably never use its own `runtime` and auth. So every Serve would have an extra runtime with default accounts and a key exchange listener, that does nothing or am I wrong?
imho we could move the `_ensureAuth` into the class scope and use the `runtime` from the current instance... or? ^^
```suggestion
```